modules/site-health/admin-site-health.php

```diff
@@ -248,9 +260,45 @@ public function status_tests( $tests ) {
 				'test'  => array( $this, 'homepage_test' ),
 			);
 		}
+		$tests['direct']['pll_simplexml'] = array(
+			'label' => esc_html__( 'SimpleXML', 'polylang' ),
+			'test'  => array( $this, 'simplexml_available' ),
+		);
```

Comment:
I propose to remove this test. It's a duplicate of the already existing WP test.

Reply:
Oh yes, because we filter here http://github.com/polylang/polylang/blob/63d3152d67c7af5fef3c1f18f4b8fa22c181ea19/modules/site-health/admin-site-health.php#L60-L60 ?